content/livesync/connector/index.textile

```diff
@@ -0,0 +1,336 @@
+
+Open a terminal and subscribe to an Ably "channel":/channels named @foo@ using "Server-Sent Events":/protocols/sse
+
+```[sh]
+curl -s -u "{{API_KEY}}" "https://realtime.ably.io/sse?channel=foo&v=1.1"
+```
+
+Run the following command to add a record to the outbox table in your PostgreSQL database. Execute a SQL statement against the database using the Railway CLI:
+
+```[sh]
+railway connect postgres
+
+railway=# INSERT INTO outbox (mutation_id, name, channel, data, headers)
+VALUES ('1', 'test', 'foo', '{}', '{}');
+```
+
+The Database Connector will detect and publish the newly inserted record as a message to the Ably channel specified in the record. In the example the channel was named @foo@. You will receive a response similar to the following over your SSE connection to Ably, indicating that you have successfully received the event:
+
+```[sh]
+id: 108GsR8ewBVHhJ@1700069266489-0
+event: message
+data: {"id":"1","connectionId":"CaqkrZ2N_0","timestamp":1700069266050,"encoding":"json","extras":{"headers":{"x-ably-models-event-uuid":"1"}},"channel":"foo","data":"{}","name":"test"}
+```
+
+h2(#options). Configuration options
+
+View the complete set of configuration options available on the connector by using @--help@, for example:
+
+```[sh]
+docker run -it --entrypoint="/adbc" ghcr.io/ably-labs/adbc:latest --help
+```
+
+<aside data-type='note'>
+<p>At least an Ably API key and the connection details of your database must be supplied to correctly configure the connector.</p>
+</aside>
+
+The following table provides descriptions for the most commonly used configuration options:
+
+|_. Option |_. Description |
+| @ADBC_ABLY_API_KEY@ | your Ably "API key":/auth#api-keys. |
+| @ADBC_POSTGRES_CONNECTION_URI@ | the full connection URI of your Postgres database. |
+| @ADBC_POSTGRES_HOST@ | your Postgres database host name as an alternative to providing the @CONNECTION_URI@. |
+| @ADBC_POSTGRES_PORT@ | your Postgres database port number as an alternative to providing the @CONNECTION_URI@. |
+| @ADBC_POSTGRES_DATABASE@ | your Postgres database name as an alternative to providing the @CONNECTION_URI@. |
+| @ADBC_POSTGRES_USER@ | your Postgres database user as an alternative to providing the @CONNECTION_URI@. |
+| @ADBC_POSTGRES_PASSWORD@ | your Postgres database user password as an alternative to providing the @CONNECTION_URI@. |
+| @--config@ | can only be specified as a CLI flag and enables you to override the path to a YAML configuration file. |
+| @ADBC_ENV@ | is an environment descriptor (either development or production). "development" pretty-prints log output with logging enabled at debug level and above. "production" logs output in JSON format with logging enabled at info level and above. |
+| @ADBC_LOG_LEVEL@ | specifies the log level to use (one of: debug, info, warn, error, fatal, panic) and overrides any presets from ADBC_ENV. |
```

Comment:
`ADBC_ENV` should be code formatted with `@ADBC_ENV@`

Reply:
Fixed in this [commit](https://github.com/ably/docs/pull/2082/commits/15a2be26866049471006c24c9e1e9094c5b53a85)